<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabQuoting">
      <attribute name="title">
       <string>&amp;Квоты</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="CTableView" name="tblClientQuoting"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFind">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="pnlDates" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chkBegDate">
            <property name="text">
             <string>Период действия с: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CDateEdit" name="edtBegDate">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblEndDate">
            <property name="text">
             <string>по: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CDateEdit" name="edtEndDate">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="chkStatus">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Статус</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cmbStatus">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Отменено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ожидание</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Активный талон</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Талон для заполнения</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Заблокированный талон</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Отказано</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Необходимо согласовать дату обслуживания</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Дата обслуживания на согласовании</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Дата обслуживания согласована</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Пролечен</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Обслуживание отложено</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Отказ пациента</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Импортировано из ВТМП</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QWidget" name="pnlButtonBox" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QDialogButtonBox" name="buttonBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="standardButtons">
             <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="chkClass">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Класс</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cmbClass"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>chkClass</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbClass</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkStatus</sender>
   <signal>clicked(bool)</signal>
   <receiver>cmbStatus</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBegDate</sender>
   <signal>clicked(bool)</signal>
   <receiver>edtBegDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBegDate</sender>
   <signal>clicked(bool)</signal>
   <receiver>edtEndDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
